string[] regs_of_space_id
---
bool result